tests/monotouch-test/MediaAccessibility/ImageCaptioningTest.cs

```diff
@@ -67,6 +67,12 @@ public void GetMetadataTagPath ()
 		[Test]
 		public void SetCaption ()
 		{
+			// looks like Apple broke something inside Xcode 12.2 Beta 2 in both sim and device
+			// NSInvalidArgumentException Reason: -[__NSCFType _getValue:forType:]: unrecognized selector sent to instance
```

Comment:
Did you file an issue?

Reply:
Here: https://github.com/xamarin/xamarin-macios/issues/9744